data/helpers.d/php

```diff
@@ -2,7 +2,7 @@
 
 # Declare the actual php version to use.
 # A packager willing to use another version of php can override the variable into its _common.sh.
-YNH_PHP_VERSION=${YNH_PHP_VERSION:-$YNH_DEFAULT_PHP_VERSION}
+YNH_PHP_VERSION=${YNH_PHP_VERSION:-${YNH_DEFAULT_PHP_VERSION:-7.0}}
```

Comment:
Another solution:

```suggestion
if [ -n "$YNH_DEFAULT_PHP_VERSION" ]; then
	YNH_PHP_VERSION=${YNH_PHP_VERSION:-YNH_DEFAULT_PHP_VERSION}
fi
```

Reply:
If you tried it (that syntax is sometimes touchy), I prefer your first proposition, on a single line.
Otherwise, I'd go for 2 lines, one for each variable to keep it clear and simple.

Reply:
In this case, if we redefine YNH_DEFAULT_PHP_VERSION with a new value, why add it to the core? I mean, let's take it out of app.py, and put it here?

Reply:
You think it would be easier to have only the definition of the variable here ?
I don't know what's going into the core about it, so I couldn't say.

In my case, I use a definition in this file for my test, I wasn't running on unstable. It worked.
But, then, to have another line with the definition of the variable would be better for clarity.

Reply:
I don't like the idea of having several location for the same thing... Another solution would store it into a file? `/etc/yunohost/default_php_version`

Reply:
Do we need to have the default version somewhere else than here ?

Reply:
Currently, it is not used in the core, so I don't know.... I'd say, no

Reply:
I feel like the only place we need to know that would be the apps scripts.
So for me, it's fine to have that info only in that file.